Sources/Xcodeproj/pbxproj.swift

```diff
@@ -336,7 +338,7 @@ public func xcodeProject(
             }
             // TODO: use identity instead
             // Construct a group name from the package name and optional version.
-            var groupName = package.manifestName // TODO: use identity instead?
+            var groupName = package.manifest.displayName // TODO: use identity instead?
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?

Reply:
Definitely the package name.  This is just the human readable name of the group.